lib/config/ui_config.dart

```diff
@@ -195,7 +195,7 @@ abstract class SksConfig {
   static const sizedBoxWidth = 5.0;
   static const radius = 8.0;
   static const innerPadding = EdgeInsets.symmetric(horizontal: 8, vertical: 4);
-  static const outerPadding = EdgeInsets.only(right: 12, bottom: 2);
+  static const outerPadding = EdgeInsets.only(right: 24, bottom: 2);
```

Comment:
did you check that this padding isn't used in any other place?

Reply:
Yes it is used in sks menu tab in SksUserButton. But I can't checkout it because sks menu is offline
![Simulator Screenshot - iPhone 16 Plus - 2024-11-29 at 20 43 03](https://github.com/user-attachments/assets/76581ca0-117e-48c2-ab9d-1c99b1778b66)

Reply:
try to comment condition that checks if menu is offline. We always have some kind of data (even old).

 ```dart
 if (!sksMenuData.isMenuOnline) {
      return const _SKSMenuLottieAnimation();
    }
```
line 61 of `sks_menu_screen.dart`